### DISCUSSION TITLE
Impossibile raggiungere il sito Connessione negata da

### DISCUSSION THREAD

**Discussion by lio60** [leokhoa/laragon#539] [Laragon's Articles & How-tos]
Buongiorno, non ho capito se ho sbagliato ma il risultato è quello del titolo, non riesco a raggiungere il sito
![image](https://github.com/leokhoa/laragon/assets/81469274/66b20fca-b879-489d-90bb-c4bb41cb812a)
cercando a caso ho cambiato la porta di apache ed ora mi dice
![image](https://github.com/leokhoa/laragon/assets/81469274/bc7f9e79-4137-402c-946a-35fea456060f)

**Comment by Pen-y-Fan** [comment]
Sorry, I can't read Italian; the screenshot shows Apache and Nginx using port 80. One will need to be changed e.g. use post 8080.

**Comment by Johnnypar** [comment]
> Buongiorno, non ho capito se ho sbagliato ma il risultato è quello del titolo, non riesco a raggiungere il sito ![image](https://user-images.githubusercontent.com/81469274/253733365-66b20fca-b879-489d-90bb-c4bb41cb812a.png) cercando a caso ho cambiato la porta di apache ed ora mi dice ![image](https://user-images.githubusercontent.com/81469274/253734077-bc7f9e79-4137-402c-946a-35fea456060f.png)

ciao, sembrerebbe che hai attivato il protocollo **SSL** dal tuo sito, ti consiglio di attivarlo anche in Laragon
-- -- --
hello, it would seem that you have activated the **SSL** protocol from your site, I recommend that you also activate it in Laragon

![test](https://github.com/leokhoa/laragon/assets/135172923/c36975ea-084c-415a-9b08-a0925024d4d0)

**Reply by lio60** [reply]
Hi Johnny and Pen-y-Fan, I apologize for not answering sooner but, setting the program as per your explanation, now mysql doesn't even start and I say now because I haven't started the program since then. Thanks for your replies Lionel.
![image](https://github.com/leokhoa/laragon/assets/81469274/0c6cc779-f34b-40d8-8905-812a4e327bb5)
